ext/mbstring/libmbfl/filters/mbfilter_htmlent.c

```diff
@@ -31,9 +31,7 @@
 #include "config.h"
 #endif
 
-#ifdef HAVE_STRING_H
 #include <string.h>
-#endif
 
 #ifdef HAVE_STRINGS_H
```

Comment:
This include can most likely be dropped.

Reply:
Indeed. Dropped.